cadence/values.yaml

```diff
@@ -57,6 +57,65 @@ server:
   affinity: {}
 
   config:
+    # clusterMetadata contains the settings of the Cadence clusters
+    # participating in a Cadence cluster group.
+    clusterMetadata:
+      # enableGlobalDomains when set to true allows creating global domains
+      # which are replicated between all clusters and whose tasks are served by
+      # the active cluster in the replication.
+      #
+      # WARNING: This value **MUST NOT** be changed after launching the Cadence
+      # cluster group.
+      enableGlobalDomain: true
+
+      # failoverVersionIncrement is the increment of each cluster version when
+      # failover happens.
+      #
+      # WARNING: For each cluster, (initialFailoverVersion +
+      # i*failoverVersionIncrement) **MUST** result in a unique value.
+      failoverVersionIncrement: 10
+
+      # masterClusterName is the name of the only cluster which can register
+      # and update a domain, although each cluster can do domain failover.
```

Comment:
It's only restricting domain registration. Updating domain can be done in any cluster as long as the domain is existing in the cluster. 

More details:
Updating domain in any cluster will work because each cluster has a unique value of `initialFailoverVersion`  so that their updating results into different `version` and they will never conflict. The updating command will be replicate across clusters and the one with largest version will finally win.

Restricting domain registration is to make sure each domain has a correct "domainUUID" across clusters. So we have to make sure there is always one masterCluster at all time.